src/v2/Apps/Fair/Components/FairCollections/FairCollection.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairCollection_collection } from "v2/__generated__/FairCollection_collection.graphql"
+import { SmallCard } from "@artsy/palette"
+import { crop } from "v2/Utils/resizer"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+const CARD_WIDTH = 263
+const CARD_LARGE_IMAGE_SIZE = 170
+const CARD_SMALL_IMAGE_SIZE = CARD_WIDTH - CARD_LARGE_IMAGE_SIZE
+const CARD_IMAGE_SIZES = [
+  CARD_LARGE_IMAGE_SIZE,
+  CARD_SMALL_IMAGE_SIZE,
+  CARD_SMALL_IMAGE_SIZE,
+]
+
+interface FairCollectionProps {
+  collection: FairCollection_collection
+}
+
+export const FairCollection: React.FC<FairCollectionProps> = ({
+  collection,
+}) => {
+  const imageUrls = collection.artworks.edges.map(
```

Comment:
We don't have strict null-checking here, but you may want to make this more defensive.

Reply:
Updated: ‧͙⁺˚*･༓☾ with more safety ☽༓･*˚⁺‧͙